lib/pluginmanager/util.rb

```diff
@@ -17,6 +19,7 @@ def self.logstash_plugin?(plugin, version = nil)
       end
     else
       dep = Gem::Dependency.new(plugin, version || Gem::Requirement.default)
+      Gem.sources = Gem::SourceList.from(options[:sources]) if options[:sources]
```

Comment:
in `LogStash::Bundler#invoke!` https://github.com/purbon/logstash/blob/13da8e2f6612c1a4af3377d9bbb1875fd0a791e6/lib/bootstrap/bundler.rb#L82 we use the `rubygems_sources` option, we should be consistent and either use `rubygems_sources` or `source` for both.